chains/superseedtestnet/metadata.yaml

```
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://sepolia-explorer.superseed.xyz/api
+    family: blockscout
+    name: Superseed Sepolia Explorer
+    url: https://sepolia-explorer.superseed.xyz/
+chainId: 53302
+displayName: Superseed Testnet
```

need deployer metadata e.g.
```suggestion
deployer:
  name: Fazleiw
  url: https://github.com/Fazleiw
displayName: Superseed Testnet
```

---

> need deployer metadata e.g.

donee, i think all good to go?